blocklib/newmod/lib/meson.build

```diff
@@ -0,0 +1,34 @@
+newmod_sources += []
+newmod_deps += []
+
+block_cpp_args = ['-DHAVE_CPU']
+
+# if cuda_dep.found() and get_option('enable_cuda')
```

Comment:
we should be able to go ahead and uncomment this for the lib if you can figure out how to switch on if newmod_cu_sources is empty - something like:
```
if cuda_dep.found() and get_option('enable_cuda') and newmod_cu_sources != []
```

don't know if that is valid meson, but something like that